pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/executor/RoundRobinScheduler.java

```diff
@@ -131,6 +131,16 @@ public int size() {
     return _ready.size() + _available.size();
   }
 
+  @Override
+  public void shutDown() {
```

Comment:
While this thread is draining the queue and closing do we still allow new OpChains to be submitted via the register method? Would it make sense to deny the new chains from being registered?

Reply:
The register won't be called in OpChainSchedulerService. I added TODO and a boolean to prevent it from happening.